orcestra_book/faq.md

```diff
@@ -0,0 +1,13 @@
+# FAQ
+
+## How to use the build environment in Jupyter lab?
+
+If you are using Jupyter notebooks/lab to run Python code on your machine, you
+can install the `orcestra_book` environment as an ipython kernel.
+
+This way you can select `orcestra_book` as kernel to run your notebooks.
+
+```bash
+conda activate orcestra_book
+ipython3 -m ipykernel install --user --name orcestra_book
+```
```

Comment:
```suggestion
```bash
conda create -n orchestra_book
conda activate orcestra_book
ipython3 -m ipykernel install --user --name orcestra_book
```
```

Reply:
The idea is to keep it as minimal as possible, so the example assumes `orcestra_book` to be present

Reply:
Then maybe mention to install `orchestra_book` into a new environment? (line 6)

```suggestion
```bash
can install the `orcestra_book` into a new environment as an ipython kernel.
```
```